src/es/components/molecules/form/Form.js

```diff
@@ -178,6 +205,17 @@ export default class Form extends Shadow() {
         padding: 0 var(--content-spacing);
         max-width: 30vw !important;
       }
+      .umbraco-forms-field-wrapper{
```

Comment:
does this no change every field even without maxlength? no side effects?

Reply:
No side effects, everything works the same